WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -166,6 +166,8 @@ def do_import
   end
 
   private def user_for_registration!(registration_row)
+    registration_row[:wca_id]&.upcase!
+    registration_row[:email]&.downcase!
```

Comment:
Do you know if we always downcase emails before putting them in our database?

Reply:
Yes, we do! https://github.com/thewca/worldcubeassociation.org/blob/835dc7d7405754488de2a0c77236afe2d87fe29a/WcaOnRails/config/initializers/devise.rb#L45-L48